pkg/asset/installconfig/platform.go

```diff
@@ -88,6 +88,7 @@ func (a *platform) queryUserForPlatform() (platform string, err error) {
 			Prompt: &survey.Select{
 				Message: "Platform",
 				Options: types.PlatformNames,
+				Help: "The platform on which the cluster will run.  For a full list of platforms, including those not supported by this wizard, see https://github.com/openshift/installer/blob/master/README.md#supported-platforms",
```

Comment:
Hmm, this might direct user's to landing page that has updated list of platforms, compared to the ones supported by the binary.

so maybe drop the link to master?

Reply:
We get occasional questions like "I thought the installer supported `${PLATFORM}`, but the wizard isn't offering it".  Linking to the list of platforms was the best way I could think of to address that in help text.  I agree that drifting versions weakens the approach somewhat, although personally I think a floaty link is better than no link at all.  And it would be great if we could pin the link or otherwise make it more obvious how to manage the float.  Maybe "new in vX.Y.Z" annotations in the master README?

Reply:
What if we drop the explicit link to supported-platform and just point people to the GitHub repo. That would prevent us from linking _directly_ to wrong set of support.

Reply:
I like the repo link.  But I just realized we have a commit ID too.  Should I link to commit-pinned docs?

Reply:
> Should I link to commit-pinned docs?
With art/ci build pipelines i think let's not depend on commits to link relation. I think repo link is goo enough for now. maybe later we can link to some public docs links.

Reply:
ok, rerolled to use the less-specific repo link in ae733b238 -> 00af9fd7a.  I tried to capture this discussion in the new commit message, if you want to take a look at that too.